doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -187,19 +187,16 @@ other. In case of CaaS some cloud infrastructure features (e.g.: HW
 management or multitenancy) are implemented by using an underlying
 **IaaS** layer.
 
-**Cloud Infrastructure Hardware Profile:** defines the behaviour,
+**Cloud Infrastructure Hardware Profile:** Defines the behaviour,
```

Comment:
we do not need to capitalize is the term is part of the sentence

Reply:
@pgoyal01 Okay, I will modify and apply ths rule

Reply:
@pgoyal01 Done!